README.md

```diff
@@ -1 +1,23 @@
 # hw01
+
+# Self Introduction
+
+### Background
+
+My name is **Yucheng Lu**, and I am originally from Shanghai, China.  I just finished my undergraduate degree in *Wake Forest University* with a major in Psychology and a minor in Japanese.  I thought the mascot of our school is funny, so here is a link to his picture.
```

Comment:
Hi, Yucheng! Your bio was very interesting, and I really enjoyed learning about you. You started off strong and already included 3 of the 5 elements in your first section; great job!